packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -381,6 +366,14 @@ function updateItem(newItem: any) {
   }
 }
 
+function renameFlist(url: string) {
+  if (url.includes("tf-official")) {
+    return url.substring(url.lastIndexOf("/") + 1, url.lastIndexOf("."));
+  }
+
+  return url.length > 40 ? url.substring(0, 40) + "..." : url;
```

Comment:
we are showing the first part of the URL which is not useful and take space from the table

Reply:
Should I remove the flist column?

Reply:
leave it. But crop the domain part and leave the flist name

Reply:
done.